### PR TITLE
UMAP at tumor purity threshold

### LINKED ISSUE #1667: Revision: UMAP at tumor purity threshold
We'll want to re-generate the UMAP at the tumor purity threshold. For this, we'll add functionality to the `transcriptomic-dimension-reduction` module to:

- Generate UMAP for tumor purity filtered data
- Add a notebook to compare that UMAP to the original UMAP made from the full data

### PR DESCRIPTION
**Pull request by sjspielman**
Closes #1667 

This PR updates the `transcriptomic-dimension-reduction` module to perform one additional UMAP on the stranded data filtered to high tumor purity samples. This PR includes a notebook with some quick before/after UMAPs, and during review we can decide the "right" place for that code to live if it's not in a notebook here!

Punchline: Broadly consistent 🎨

### CODE REVIEW

**Comment by sjspielman** [comment]
Noting that this is now passing CI at least up through the relevant module.

File: analyses/transcriptomic-dimension-reduction/ci-dimension-reduction-plots.sh [hunk 1]
```diff
@@ -41,6 +41,16 @@ Rscript --vanilla scripts/run-dimension-reduction.R \
   --neighbors 2 \
   --skip_tsne
 
+# Run at tumor purity threshold
```
Comment (jaclyn-taroni) [inline analyses/transcriptomic-dimension-reduction/ci-dimension-reduction-plots.sh:44]: Missing `--log2_transform` maybe? If only for completeness.

**Review by jaclyn-taroni — APPROVED** [review]
👍🏻 I say we just get this in since it's already written and then come up with a broader strategy for #1665

File: analyses/transcriptomic-dimension-reduction/ci-dimension-reduction-plots.sh (hunk 1 above)
Comment (sjspielman) [reply analyses/transcriptomic-dimension-reduction/ci-dimension-reduction-plots.sh:44]: Done in https://github.com/AlexsLemonade/OpenPBTA-analysis/pull/1668/commits/4d6bd6f9e187df984e81222b73f238476e166b0a

**Pull request merged**